Various class/unit additions

sm453 opened the pull request:
Apologies for the (very long) delay. Feedback very welcome.

Commit:
mj_to_kg: added the mmegajoule per kilogram unit

Commit:
mj_to_kg: removed erroneous namespace

jwja2 committed:
Merge pull request #2 from cambridge-cares/mj_to_kg

Approved merged of MJ/kg addition.

Commit:
IIB_units: added all units required

Commit:
IIB_units: added dimensions for all that have them

jwja2 committed:
Merge pull request #3 from cambridge-cares/IIB_units

IIB units

Commit:
trs53_iriupdate: Changes to namespaces of additional units

msff2 committed:
Merge pull request #4 from cambridge-cares/trs53_iriupdate

trs53_iriupdate: Changes to namespaces of additional units

jb2197 committed:
Merge branch 'master' into develop

jb2197 committed:
136-dev-add-custom-units-to-ontouom: added unit pound_sterling_per_litre and kilogram_per_litre_per_minute to support concepts RunMaterialCost and SpaceTimeYield as part of PerformanceIndicator of a ReactionExperiment.

jb2197 committed:
Merge pull request #5 from cambridge-cares/136-dev-add-custom-units-to-ontouom

136-dev-add-custom-units-to-ontouom: added unit pound_sterling_per_li…

jb2197 committed:
136-dev-add-custom-units-to-ontouom: added custom unit to support expressing peak area of a chromatogram reading.

markushofmeister committed:
dev-add-concepts-and-units-for-OntoEMS: Included MassOfSubstanceConcentration quantity and amended units for om:VolumetricFlowRate and om:Length to represent all relevant quantities and units required for OntoEMS

markushofmeister committed:
Merge pull request #6 from cambridge-cares/dev-add-concepts-and-units-for-OntoEMS

Include OntoEMS concepts

markushofmeister committed:
Merge pull request #7 from cambridge-cares/develop

Merge OntoEMS additions from Develop into Master

markushofmeister committed:
dev-add-concepts-and-units-for-OntoEMS: Add nanogramPerCubicMetre as new unit

markushofmeister committed:
Merge pull request #8 from cambridge-cares/dev-add-concepts-and-units-for-OntoEMS

Add nanogramPerCubicMetre as new unit

markushofmeister committed:
dev-OntoBuiltEnv-extension: Added Quantity and Unit to represent property values per quare metre

markushofmeister committed:
Merge pull request #9 from cambridge-cares/dev-OntoBuiltEnv-extension

dev-OntoBuiltEnv-extension

jb2197 committed:
Merge branch 'master' into 136-dev-add-custom-units-to-ontouom

jb2197 committed:
136-dev-add-custom-units-to-ontouom: changed namespace for added unit as requested.

jb2197 committed:
136-dev-add-custom-units-to-ontouom: added gramPerMole unit division.

jb2197 committed:
136-dev-add-custom-units-to-ontouom: added millilitrePerMinute-Time unit division.

jb2197 committed:
Merge pull request #10 from cambridge-cares/136-dev-add-custom-units-to-ontouom

136 dev add custom units to ontouom

jb2197 committed:
Merge branch 'HajoRijgersberg:master' into master

jb2197 committed:
191-dev-pound-sterling-per-kilogram: added unit division poundSterlingPerKilogram.

jb2197 committed:
Merge pull request #11 from cambridge-cares/191-dev-pound-sterling-per-kilogram

191-dev-pound-sterling-per-kilogram: added unit division poundSterlin…

sm453 committed:
master: Added symbol to euro unit.

sm453 committed:
master: Swapped symbol and alternative symbol for kilowatt hour.

sm453 committed:
Merge branch 'HajoRijgersberg:master' into master

Abyssus1010 committed:
1504-dev-shift-soundpressurelevel-and-units-to-om: add SoundPressureLevel as subclass of om:Quantity, add decibelsRelativeToFullScale as a om:SingularUnit, add SoundPressureLevelUnit as subclass of om:Unit, add om:decibel and decibelsRelativeToFullScale as a collection under SoundPressureLevelUnit

Abyssus1010 committed:
1504-dev-shift-soundpressurelevel-and-units-to-om: modify Decibels Relative to Full Scale definitions

Abyssus1010 committed:
Merge pull request #12 from cambridge-cares/1504-dev-shift-soundpressurelevel-and-units-to-om

1504-dev-shift-soundpressurelevel-and-units-to-om: add SoundPressureL…

mingchuantian812 committed:
Added volt-ampere and volt-ampere reactive an new units, as well as
killovolt-ampere (kVA) and killovolt-ampere reactive (kVAR) for power
system.

mingchuantian812 committed:
Re-organised newly added units. Moved wrongly placed triples to the
right places. Corrected mistakes.

mingchuantian812 committed:
Corrected "hasDimension" property for "Volt-ampere", "Volt-ampere
reactive", and their multiples.

mingchuantian812 committed:
Corrected symbols for Megavolt-ampere and Megavolt-ampere Reactive from "mVA" and "mVAR" to "MVA" and "MVAR".

sm453 committed:
dev-concentration-units: Added microgram per cubic metre units/classes (in OM namespace!).

sm453 committed:
Merge pull request #14 from cambridge-cares/dev-concentration-units

dev-concentration-units: Added microgram per cubic metre units/classe…

sm453 committed:
dev-micro-character: Replaced micro character with HTML representation in microgram per cubic metre unit.
This fixes some display issues. NB HTML representation is already being used for other characters, e.g. degree.

sm453 committed:
Merge pull request #15 from cambridge-cares/dev-micro-character

dev-micro-character: Replaced micro character with HTML representatio…

mingchuantian812 committed:
Merge pull request #13 from cambridge-cares/1512-dev-add-kVAR-unit-for-OntoPOWSYS

1512-dev-add-kVAR-unit-for-OntoPOWSYS: Add Volt-Ampere (VA), Volt-Ampere Reactive (VAR), kVA, kVAR, MVA, and MVAR in OM-2 for power system.

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add cubicMeterPerHour

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add litrePerMinute

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add pound-ForcePerSquareInch

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add missing / symbol

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add litrePerMinute as part of VolumetricFlowRateUnit collection

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: edit label of pound-ForcePerSquareInch

Abyssus1010 committed:
1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om: add labels for litrePerMinute, pound-ForcePerSquareInch, cubicMetrePerHour

Abyssus1010 committed:
Merge pull request #16 from cambridge-cares/1522-dev-add-litreperminute-pound-forcepersquareinch-and-cubicmetreperhour-to-om

1522 dev add litreperminute pound forcepersquareinch and cubicmetreperhour to om

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: update previously introduced units to OM prefix and to be consistent with OM formatting

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: move AmountOfMoneyPerEnergy concepts from OntoUOM to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: move AmountOfMoneyPerTimeInterval concepts from OntoUOM to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: move Concentration concept from OntoUOM to OM and remove duplicate MassOfSubstanceConcentration concepts

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: move HeatFluxDensityUnit and wattHourPerSquareMetrePerDay from OntoUOM to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add SpecificYield concepts to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add kilojoulePerKelvinKilogram to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add massPerEnergy-Dimension and tonnePerMegawattHour to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add EnergyPerArea concepts to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add kilowattHourPerYear, megawattHourPerYear, and gigawattHourPerYear to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add kilowattHourPerCubicMetre to OM

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add dimension information to EnergyPerArea quantity and decapitalise MassOfSubstanceConcentration-Dimension

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add in definition of water level height units

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: update AmountOfSubstanceConcentration to be subclass of Concentration

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add additional flow rate unit definitions

YiKaiTsai committed:
1527-dev-move-ontouom-units-to-om: add nanogramPerCubicMetre definition

YiKaiTsai committed:
Merge pull request #17 from cambridge-cares/1527-dev-move-ontouom-units-to-om

1527 dev move ontouom units to om

Abyssus1010 committed:
1607-dev-add-missing-end-tags-for-several-unitdivision-elements: Add missing or incomplete end tags for several UnitDivision elements

Abyssus1010 committed:
Merge pull request #18 from cambridge-cares/1607-dev-add-missing-end-tags-for-several-unitdivision-elements

1607-dev-add-missing-end-tags-for-several-unitdivision-elements: Add …

markushofmeister committed:
hotfix-resolve-parsing-issues: Fix two RDF/XML syntax issues which prevent rdflib and RemoteStoreClient from parsing the file

markushofmeister committed:
Merge pull request #19 from cambridge-cares/hotfix-resolve-parsing-issues

hotfix-resolve-parsing-issues: Fix two RDF/XML syntax issues

Abyssus1010 committed:
1763-dev-add-revolution-and-units-to-om: Add revolution per minute unit

sm453 committed:
dev-merge-head-rep: Merged branch 'master' from git@github.com:HajoRijgersberg/OM.git into dev-merge-head-rep.

# Conflicts:
#	om-2.0.rdf
# Nothing to resolve.

sm453 committed:
Merge pull request #22 from cambridge-cares/dev-merge-head-rep

Dev merge head rep

Abyssus1010 committed:
Merge pull request #20 from cambridge-cares/1763-dev-add-revolution-and-units-to-om

1763-dev-add-revolution-and-units-to-om: Add revolution per minute unit

HajoRijgersberg commented:
Hey, no problem. My apologies for my late reply! And thanx for your contribution. I just took a quick and it looks fine. I'll have a deeper/closer look soon, I intend to that the coming week. Let's keep each other updated.

Abyssus1010 committed:
1791-dev-add-particle-concentration-and-particle-count-per-cubic-centimetres: Added particles unit instance, Particle Concentration class, particle count per cubic centimetre

Abyssus1010 committed:
1791-dev-add-particle-concentration-and-particle-count-per-cubic-centimetres: Slight modification

particles, particleConcentration and particleCountPerCubicCentimetre are renamed as particleNumber, particleNumberConcentration and particleNumberPerCubicCentimetre

Abyssus1010 committed:
Merge pull request #23 from cambridge-cares/1791-dev-add-particle-concentration-and-particle-count-per-cubic-centimetres

1791-dev-add-particle-concentration-and-particle-count-per-cubic-cent…

HajoRijgersberg commented:
My deep apologies that I have been late. And I didn't realize that such a lot of changes were involved, which is great!
So, I'm trying to study all, and by a kind of coincidence I started with the latest changes, about particle numbers, but I think that these are no units but quantities. Could you please adapt that? In the meantime I will continue the other changes.
Sorry for the inconvenience, also for having to change! But many thanx in advance!

Abyssus1010 committed:
1823-dev-add-weight-percent-unit: Add weight percent unit under Mass Fraction Unit

Abyssus1010 committed:
1823-dev-add-weight-percent-unit: add pH Level as a subclass of acidity

Abyssus1010 committed:
1823-dev-add-weight-percent-unit: Modification to om:Acidity, remove pHLevel and weightPercent

Abyssus1010 committed:
1823-dev-add-weight-percent-unit: Change alternativeSymbol to alternativeLabel

Abyssus1010 committed:
Merge pull request #25 from cambridge-cares/1823-dev-add-weight-percent-unit

1823-dev-add-weight-percent-unit: Add weight percent unit under Mass …

simonrihm committed:
added elementary charge unit

sm453 committed:
Merge branch 'HajoRijgersberg:master' into master

sm453 commented:
> ... about particle numbers, but I think that these are no units but quantities.

Wouldn't this imply that all derived units, like this one
`<om:UnitDivision rdf:about="&om;particleNumberPerCubicCentimetre"/>`
are also quantities?

I guess one could argue that particle number is a quantity that is dimensionless, and then only define units of e.g. a particle number concentration like "1/cm^3", which would be technically correct, but that would be a shame (because one would lose information). And I'm not sure everyone working in that field would necessarily agree with that...

(Apologies for the _extremely long_ delay!)
